digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/CalibrationPointDAO.scala

```diff
@@ -117,6 +129,12 @@ object CalibrationPointDAO {
       """.execute
   }
 
+  def removeAllCalibrationPoints(projectLinkIds: Set[Long]) = {
```

Comment:
Might be good to handle the special case when the projectLinkIds is an empty Set.